components/LoginModal.vue

```diff
@@ -30,20 +25,20 @@
             v-model="rememberMe"
             name="remember-me"
             label="Remember me"
-            class="form__element form__checkbox"
+            class="form__element remember-me"
           />
           <span v-if="userError.login" class="login-error-message">
             {{ userError.login.message }}
           </span>
 
           <SfButton
             type="submit"
-            class="sf-button--full-width form__submit"
+            class="sf-button form__submit login-button"
             data-testid="log-in-button"
             :disabled="loading"
           >
             <SfLoader :class="{ loader: loading }" :loading="loading">
-              <div>{{ $t("Login") }}</div>
+              <div>Log In</div>
```

Comment:
use i18n lang helper instead of hardcoding the text

Reply:
Resolved